pkg/action/install.go

```diff
@@ -0,0 +1,67 @@
+package action
+
+import (
+	"time"
+
+	"github.com/helmwave/helmwave/pkg/kubedog"
+	"github.com/helmwave/helmwave/pkg/plan"
+	"github.com/urfave/cli/v2"
+)
+
+type Install struct {
+	dog      *kubedog.Config
+	plandir  string
+	parallel bool
+	kubedog  bool
```

Comment:
kubedogEnabled?

Reply:
kubedog means in install context as kubedogEnabled